build.sbt

```diff
@@ -61,6 +61,7 @@ lazy val schema =
         "io.circe"       %%% "circe-testing"                   % circeVersion               % Test,
         "edu.gemini"     %%% "lucuma-core"                     % lucumaCoreVersion,
         "edu.gemini"     %%% "lucuma-core-testkit"             % lucumaCoreVersion          % Test,
+        "edu.gemini"     %%% "lucuma-sso-frontend-client"      % lucumaSsoVersion,
```

Comment:
I added this just to pick up the circe codec for `User` so that I could in turn provide a codec for `TimeChargeCorrection`.  Is this ill advised?  Maybe I should just supply a mapping.

Reply:
I think this is ok. We could factor out another tiny module with just the codec at some point. Because the codec is used for encoding/decoding the JWT payload it has to be very stable and has to be owned by SSO I think.